jaxrs-tck/src/main/resources/ee/jakarta/tck/ws/rs/signaturetest/jakarta.ws.rs.sig_3.2.0

```diff
@@ -504,7 +504,6 @@ meth public static jakarta.ws.rs.client.Client newClient()
 meth public static jakarta.ws.rs.client.Client newClient(jakarta.ws.rs.core.Configuration)
 meth public static jakarta.ws.rs.client.ClientBuilder newBuilder()
 supr java.lang.Object
-hcls CreateErrorMessageAction
```

Comment:
I do not think 3.2.0 signature file should be changed for 5.0.
Instead, a new 5.0 should have been created.

Reply:
It is not necessary to do so now, though. It would need to be created after all the changes for 5.0.

Reply:
Thank you, Jan, for this clarification! I have undone all changes to that file. 🙂